pkg/subctl/cmd/export.go

```diff
@@ -77,9 +72,9 @@ func exportService(cmd *cobra.Command, args []string) {
 
 	utils.ExitOnError("Error creating RestMapper for ServiceExport", err)
 
-	if serviceNamespace == "" {
-		if serviceNamespace, _, err = clientConfig.Namespace(); err != nil {
-			serviceNamespace = "default"
+	if namespace == "" {
```

Comment:
Could also make similar changes to https://github.com/submariner-io/submariner-operator/blob/devel/cmd/subctl/export.go please.

Reply:
We can do that next release when we resume the restructuring.